packages/sudoku-utils/src/utils/cell-clues.ts

```diff
@@ -159,3 +166,201 @@ export function getCellCluesToDraw(clue: Cellclue): Cellclue[] {
 
   return drawClues;
 }
+
+/*Checks the inputted digits against the standard constraint logic for the cell clue
+and returns any cells that have errors*/
+export function verifyCellClue(
+  cellclue: Cellclue,
+  solution: CellValues,
+  clues: EditorHistoryStep
+): Position[] {
+  if (!(cellclue.nonStandard ?? false)) {
```

Comment:
Do an early return here to mitigate indent level
```suggestion
  if ((cellclue.nonStandard ?? false)) {
    return []
  }
```

Reply:
done